dev-util/spec-cleaner/files/spec-cleaner-1.0.6-pytest-runner.patch

```diff
@@ -0,0 +1,10 @@
+--- a/setup.py 2018-03-19 15:23:46.128000000 +0000
++++ b/setup.py 2018-03-19 15:24:26.920000000 +0000
+@@ -43,7 +43,6 @@
+     platforms=['Linux'],
+     keywords=['SUSE', 'RPM', '.spec', 'cleaner'],
+ 
+-    setup_requires=['pytest-runner'],
+     tests_require=['pytest', 'pytest-cov'],
```

Comment:
I'm going to guess you want to remove `pytest-cov` from here if you want to run it via setup.py. Alternatively, `py.test -v || die` would also work.

Reply:
Not sure what you are saying here. Everything looks fine to me, and I also don't get any warning

Reply:
```
 * python3_4: running distutils-r1_run_phase python_test
/usr/bin/python3.4 setup.py test
running pytest
Searching for pytest-cov
Reading https://pypi.python.org/simple/pytest-cov/
Download error on https://pypi.python.org/simple/pytest-cov/: [Errno -2] Name or service not known -- Some packages may not be found!
Couldn't find index page for 'pytest-cov' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.python.org/simple/
Download error on https://pypi.python.org/simple/: [Errno -2] Name or service not known -- Some packages may not be found!
No local packages or working download links found for pytest-cov
error: Could not find suitable distribution for Requirement.parse('pytest-cov')
```

Told you you need to strip it.

Reply:
What do you run to get this? Why doesnt this happen for me? oO